cmd/utils/flags.go

```diff
@@ -312,6 +312,14 @@ var (
 		Name:  "statedb.cache.redis.cluster",
 		Usage: "Enables cluster-enabled mode of redis trie node cache",
 	}
+	TrieNodeCacheRedisPublishBlockFlag = cli.BoolFlag{
+		Name:  "statedb.cache.redis.publish",
```

Comment:
Don't we have `kes` namespace in the configuration?

Reply:
I can't find `kes.~~`.